index.html

```diff
@@ -23,6 +23,10 @@ <h3>jeszcze niższego poziomu nagłówek</h3>
         ad? Nam recusandae, consectetur doloremque maiores id ullam laudantium corrupti iure tenetur eveniet, officia
         voluptates!</p>
 
```

Comment:
I think, here have to change this paragraph

Reply:
Yes, You are right!